e2e/run_tests.js

```diff
@@ -161,7 +161,22 @@ function printMessage(testFiles, index) {
 
     // Log which files were being tested
     console.log(chalk.magenta.bold(`${invert ? 'All Except --> ' : ''}${testStage}${stage && withGroup ? '| ' : ''}${testGroup}`));
-    console.log(chalk.magenta(`(Testing ${index + 1} of ${testFiles.length})  - `, testFile));
+    console.log(chalk.magenta(`(Testing ${overallIndex + 1} of ${testFiles.length})  - `, testFile));
+    if (process.env.CI_BASE_URL) {
+        console.log(chalk.magenta(`Testing ${currentIndex}/${lastIndex} in "${process.env.CI_BASE_URL}" server`));
+    }
+}
+
+function getTestFilesIdentifier(numberOfTestFiles) {
+    const {part, of} = argv;
+    const multiplier = Math.round(numberOfTestFiles / of);
```

Comment:
I think this needs to be a ceiling,
numberOfTestFiles=10
part=3
of=3
multiplier = Math.ceil(10/3) = 4
start = (3 - 1) * 4 = 8
end = 8 + 4 = 12
end = 12 < 10 ? 12 : 10 = 10

Otherwise if it's round,
numberOfTestFiles=10
part=3
of=3
multiplier = Math.round(10/3) = 3
start = (3 - 1) * 3 = 6
end = 6 + 3 = 9
end = 9 < 10 ? 9 : 10 = 9

will be missing 1 file?

```suggestion
    const multiplier = Math.ceil(numberOfTestFiles / of);
```

Reply:
Nice catch Joseph! You're correct, will update this. 
Also, I missed to add the default value of `1` to `part` and `of` so that running the script without passing such args (ex. `node run_tests.js`) will still work as expected.

Reply:
Done, PTAL. Thanks!